test/powershell/Host/Logging.Tests.ps1

```diff
@@ -201,34 +201,33 @@ Describe 'Basic os_log tests on MacOS' -Tag @('CI','RequireSudoOnUnix') {
         }
     }
 
-    ## Logging seems broken on macOS 10.13. It works fine on macOS 10.12.6.
-    ## Travis CI updated macOS to 10.13.3 (kernel 17.4) and the logging tests start to fail.
-    # It 'Verifies basic logging with no customizations' -Skip:(!$IsSupportedEnvironment) {
-    It 'Verifies basic logging with no customizations' -Pending {
+    It 'Verifies basic logging with no customizations' -Skip:(!$IsSupportedEnvironment) {
         $configFile = WriteLogSettings -LogId $logId
-        & $powershell -NoProfile -SettingsFile $configFile -Command '$env:PSModulePath | out-null'
+        $testPid = & $powershell -NoProfile -SettingsFile $configFile -Command '$PID'
 
-        Export-PSOsLog -After $after -Verbose | Set-Content -Path $contentFile
-        $items = Get-PSOsLog -Path $contentFile -Id $logId -After $after -TotalCount 3 -Verbose
+        # Made tests more reliable
+        Start-Sleep -Milliseconds 500
```

Comment:
I've created issue https://github.com/PowerShell/PowerShell/issues/7462 to address the Start-Sleep workaround as well as the possible false-positive in filtering test below.

Reply:
Thanks